rln/tests/poseidon_tree.rs

```diff
@@ -143,36 +127,45 @@ mod pmtree_test {
 
             Ok(())
         }
+
+        fn put_batch(&mut self, subtree: HashMap<DBKey, Value>) -> Result<()> {
+            self.0.extend(subtree);
+            Ok(())
+        }
     }
 
     // We implement Database for sled DB, an on-disk database
     struct SledDB(Sled);
 
     impl Database for SledDB {
-        fn new(dbpath: &str) -> Result<Self> {
-            if Path::new(dbpath).exists() {
-                match fs::remove_dir_all(dbpath) {
+        type Config = DBConfig;
```

Comment:
This config would change per type. Please create a SledDbConfig, and refer to sled's docs about usage with the `sled::open` fn

Reply:
Used [`sled::Config`](https://docs.rs/sled/latest/sled/struct.Config.html) instead of creating `SledDbConfig` as discussed internally. [a46da10](https://github.com/vacp2p/zerokit/pull/125/commits/a46da10a2ca1cd76a73d66dc8ff841f980d2cb5f)